indico/modules/events/editing/client/js/editing/timeline/ReviewComment.jsx

```diff
@@ -0,0 +1,87 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2023 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+import {Form as FinalForm} from 'react-final-form';
+import {useDispatch} from 'react-redux';
+import {Button, Form} from 'semantic-ui-react';
+
+import {FinalSubmitButton, FinalTextArea} from 'indico/react/forms';
+import {Translate} from 'indico/react/i18n';
+
+import {RevisionType} from '../../models';
+
+import {modifyRevisionComment} from './actions';
+import {blockPropTypes} from './util';
+
+import './ReviewComment.module.scss';
+
+export default function ReviewComment({block, canEdit}) {
+  const [editFormOpen, setEditFormOpen] = useState(false);
+  const dispatch = useDispatch();
+
+  const handleSubmit = async formData => {
+    const rv = await dispatch(modifyRevisionComment(block.reviewURL, formData));
```

Comment:
is it a bad practice to reuse this `modifyRevisionComment` action?

Reply:
if you changed its name it'd be fine. however, I'd just duplicate it with a new name considering how simple it is (one line). `deleteRevisionComment` and `resetReviews` are also exactly the same with different names...